packages/simcore-sdk/src/simcore_sdk/node_ports_common/filemanager.py

```diff
@@ -279,7 +281,32 @@ async def upload_file(
         if not upload_link:
             raise exceptions.S3InvalidPathError(s3_object)
 
-        e_tag = await _upload_file_to_link(session, upload_link, local_file_path)
+        if await is_r_clone_available(r_clone_settings):
+            if store_id != SIMCORE_LOCATION:

```

Comment:
this is a bit weird. since I do not see this exception in the else case. Should it not go to the else case if ```store_id!=SIMCORE_LOCATION```??

Reply:
I guess it is better to always upload instead of showing an error.